src/services/EdFi.DataManagementService.Api.Tests.E2E/Features/Resources/ReadResourcesValidation.feature.disabled

```diff
@@ -0,0 +1,56 @@
+# This is a rough draft feature for future use.
+@ignore
+Feature: Resources "Read" Operation validations
+
+        Background:
+            Given the Data Management Service must receive a token issued by "http://localhost"
+              And user is already authorized
+              And request made to "/ed-fi/absenceEventCategoryDescriptors" with
+                  """
+                    {
+                        "codeValue": "Sick Leave",
+                        "description": "Sick Leave",
+                        "effectiveBeginDate": "2024-05-14",
+                        "effectiveEndDate": "2024-05-14",
+                        "namespace": "uri://ed-fi.org/AbsenceEventCategoryDescriptor",
+                        "shortDescription": "Sick Leave"
+                    }
+                  """
+             Then it should respond with 201
+
+        @ignore
+        Scenario: Verify existing resources can be retrieved successfully
+             When a GET request is made to "/ed-fi/absenceEventCategoryDescriptors"
+             Then it should respond with 200
+
+        @ignore
+        Scenario: Verify retrieving a single resource by ID
+        # Replace Endpoint with the required value, this is just an example to make sure Descriptors are working fine
+             When a GET request is made to "/ed-fi/absenceEventCategoryDescriptors/16435704274945eeae6c6c45b281aad3"
+             Then it should respond with 200
+
+        @ignore
+        Scenario: Verify response contains expected data
+        # Replace Endpoint with the required value, this is just an example to make sure Descriptors are working fine
+             When a GET request is made to "/ed-fi/absenceEventCategoryDescriptors/16435704274945eeae6c6c45b281aad3"
+             Then it should respond with 200
+              And the response body is
+                  """
+                    {
+                        "codeValue": "Jury duty",
+                        "description": "Jury duty",
+                        "namespace": "uri://ed-fi.org/AbsenceEventCategoryDescriptor",
+                        "shortDescription": "Jury duty"
+                    }
+                  """
+
+        @ignore
+        Scenario: Verify response code 404 when ID does not exist
+             When a GET request is made to "/ed-fi/absenceEventCategoryDescriptors/123123123123"
+             Then it should respond with 404
+
+        @ignore
+        Scenario: Verify empty array when there are not records on GET All
+             When a GET request is made to "/ed-fi/absenceEventCategoryDescriptors"
```

Comment:
The background above creates a record, so this will never be zero... unless you delete that record. Suggestion: go ahead and look for 1 record, instead of deleting.